ts/features/euCovidCert/screens/EuCovidCertRevokedScreen.tsx

```diff
@@ -1,17 +1,56 @@
 import * as React from "react";
 import { connect } from "react-redux";
 import { Dispatch } from "redux";
-import { H1 } from "../../../components/core/typography/H1";
+import { Image } from "react-native";
+import { View } from "native-base";
 import { GlobalState } from "../../../store/reducers/types";
+import { InfoScreenComponent } from "../../../components/infoScreen/InfoScreenComponent";
+import I18n from "../../../i18n";
+import revokedImage from "../../../../img/features/euCovidCert/certificate_revoked.png";
+import Markdown from "../../../components/ui/Markdown";
+import { Link } from "../../../components/core/typography/Link";
+import { openWebUrl } from "../../../utils/url";
 import { BaseEuCovidCertificateLayout } from "./BaseEuCovidCertificateLayout";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
-  ReturnType<typeof mapStateToProps>;
+  ReturnType<typeof mapStateToProps> & { revokeInfo?: string };
 
-const EuCovidCertRevokedScreen = (_: Props): React.ReactElement => (
+// FIXME update the link when provided
```

Comment:
this PR lgtm
Could you add the story link after the FIXME message?
If there is no story about it, we can create a dedicated one to avoid to forget this action

Reply:
We can use `euCovidCertificateUrl` in `ts/urls.ts`

Reply:
Fixed as suggested in d3f5acf